entrypoint.sh

```diff
@@ -18,6 +18,16 @@ sh -c "git config --global credential.username $GITLAB_USERNAME"
 sh -c "git config --global core.askPass /cred-helper.sh"
 sh -c "git config --global credential.helper cache"
 sh -c "git remote add mirror $*"
+if [[ -n "${REMOVE_BRANCH}" ]]; then # Check if variable exists 
+   if [[ "${REMOVE_BRANCH}" == "true" ]]; then 
```

Comment:
Why put these on two lines? Why not put them on a single line like:
```
if [[ -n "${REMOVE_BRANCH}" ]] && [[ "${REMOVE_BRANCH}" == "true" ]]; then
```

Reply:
This is a matter of taste but I am fine with changing it.

Reply:
Since there is no `else` statement, it just seems like it should be combined.